04.ls/ls.rb

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: true
+
+COLUMNS = 3
+
+def main
+  created_file_names = file_names
```

Comment:
```suggestion
  file_names = file_names
```

変数名の方をシンプルにしたいですね。ということは・・・？
ヒント https://qiita.com/jnchito/items/fb438aa0786bb3cbf919

Reply:
メソッド名を`create_file_names`に変更して変数名をシンプルにしました。